sdk/daml-script/daml3/internal/Daml/Script/Internal/Questions/Submit/Error.daml

```diff
@@ -25,6 +25,7 @@ data UpgradeErrorType
         keyOpt : Optional (AnyContractKey, [Party])
   | DowngradeDropDefinedField with
         expectedType : Text
+        fieldIndex : Int
```

Comment:
Should the `actualValue: Value` field be exposed to daml-script?

Reply:
I don't think it'd be particularly useful, I think what is useful would be the type and perhaps the field name if possible

Reply:
Agreed that information would be nice. The problem is it is not immediately available at the error call sites - SValue's are coming from the ledger and so we need to fold over type data to recover specific values. Discussing this earlier with @remyhaemmerle-da, this might be something we want to leave for now? Perhaps this is something we should discuss further on Wednesday?